src/components/Landing.js

```diff
@@ -147,24 +150,25 @@ const Landing = () => (
         <div
           css={{
             position: "relative",
-            paddingTop: "110px",
             textAlign: "right",
-            marginRight: "80px"
+            display: "flex",
+            alignItems: "center",
+            "@media(max-width: 820px)": {
+              display: "none"
+            }
           }}
         >
           <img
             src={landingImg}
             css={{
               width: "100%",
               maxWidth: "750px",
-              "@media(max-width: 820px)": {
-                display: "none"
-              }
+              display: "block"
```

Comment:
```suggestion
              position: "relative",
              top: "50%",
              transform: "translateY(-50%)",
```

This will vertically center it

Reply:
^Think you had this there before

Reply:
But if you decrease the padding in the div on top then it will be vertically centred!